tests/unit/modules/test_file.py

```diff
@@ -235,7 +235,10 @@ def setup_loader_modules(self):
                     'grains': {},
                 },
                 '__grains__': {'kernel': 'Linux'},
-                '__utils__': {'files.is_text': MagicMock(return_value=True)},
+                '__utils__': {
+                    'files.is_binary': MagicMock(return_value=False),
+                    'files.get_encoding': MagicMock(return_value=None)
```

Comment:
You're mocking thiis to return `None`, but that function never actually returns `None`, might that cause problems in terms of the accuracy of our testing?

In addition, I'm not sure it's a great idea to mock these functions at the module level, rather than just patching in individual tests, unless a given function is used in a lot of tests and needs to be consistently mocked with the same result.